internal/dnfjson/cache.go

```diff
@@ -1,46 +1,194 @@
 package dnfjson
 
 import (
+	"fmt"
 	"io/fs"
 	"os"
 	"path/filepath"
+	"sort"
+	"time"
+
+	"github.com/gobwas/glob"
 )
 
+// A collection of directory paths, their total size, and their most recent
+// modification time.
+type pathInfo struct {
```

Comment:
Would it be more intuitive to name this something like `repoCacheEntry` or something like that? The fact that it is (a collection of) path(s) is not really the point is it?

Reply:
Yes!